File: src/main/java/nl/guno/intellij/savechangelisttopatches/MessageResources.java
package nl.guno.intellij.savechangelisttopatches;

import java.util.ResourceBundle;

import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.PropertyKey;

import com.intellij.CommonBundle;

public final class MessageResources {
    @NonNls
    private static final String BUNDLE_NAME = "nl.guno.intellij.savechangelisttopatches.MessageResources";
    private static final ResourceBundle BUNDLE = ResourceBundle.getBundle(BUNDLE_NAME);

    private MessageResources() {
    }

    @NotNull
    public static String message(@PropertyKey(resourceBundle = BUNDLE_NAME) @NotNull String key, @NotNull Object... params) {
        return CommonBundle.message(BUNDLE, key, params);

    }
}

File: src/main/kotlin/nl/guno/intellij/savechangelisttopatches/MessageResources.kt
package nl.guno.intellij.savechangelisttopatches

import java.util.ResourceBundle

import org.jetbrains.annotations.NonNls
import org.jetbrains.annotations.PropertyKey

import com.intellij.CommonBundle

object MessageResources {
    @NonNls
    private const val BUNDLE_NAME = "nl.guno.intellij.savechangelisttopatches.MessageResources"
    private val BUNDLE = ResourceBundle.getBundle(BUNDLE_NAME)

    fun message(@PropertyKey(resourceBundle = BUNDLE_NAME) key: String, vararg params: Any): String {
        return CommonBundle.message(BUNDLE, key, *params)

    }
}
